ssv/runner.go

```diff
@@ -196,7 +196,7 @@ func (b *BaseRunner) basePartialSigMsgProcessing(
 // didDecideCorrectly returns true if the expected consensus instance decided correctly
 func (b *BaseRunner) didDecideCorrectly(prevDecided bool, decidedMsg *qbft.SignedMessage) (bool, error) {
 	decided := decidedMsg != nil
```

Comment:
I suggest to write it differently, this is how we write stuff lie this in go and I also think is more readable and easier to reason about
```golang
if decidedMsg == nil {
 return false, nil
}
if b.State.RunningInstance == nil {
 return false, errors.New("decided wrong instance")
}

if decidedMsg.Message.Height != b.State.RunningInstance.GetHeight() {
 return false, errors.New("decided wrong instance")
}
```

Reply:
let me know what you guys think, I know its not a part of this PR but I couldn't help but see this
@MatheusFranco99 , @GalRogozinski , @moshe-blox . its ok also to decide not to do it here.

Reply:
I 100% agree. @GalRogozinski what do you think?